### DISCUSSION TITLE
How to calculate the returned distance without querying milvus

### DISCUSSION THREAD

**Discussion by peak-xu** [milvus-io/milvus#27569] [Q&A and General discussion]
How to calculate the returned distance without querying milvus

**Comment by yhmo** [comment]
I don't understand. If you didn't query milvus, how can it return distance for you?

**Reply by peak-xu** [reply]
Our code sets the distance to a certain value without returning data. Now we want to evaluate the appropriate distance setting. Do you have any good solutions?

**Reply by peak-xu** [reply]
If there is no other way, I can only loop to query milvus to get distance

**Reply by yhmo** [reply]
Ok, you are using [range search](https://milvus.io/docs/within_range.md#Conduct-a-Range-Search)
In my opinion, the "appropriate distance" is not predictable. If the distance value returns an empty result, just increase the distance to query again.
